### PR TITLE
[Api/#58] 회원탈퇴 api 연동

### PR DESCRIPTION
**Pull request by SebellKo**
## 📌 작업 내용

- [x] 회원탈퇴 api 연동
- [x] 로그아웃 기능 구현

회원탈퇴 api 를 연동해 기능구현했습니다.
추가로 로그아웃 기능을 구현해 회원탈퇴 완료시 세션 스토리지를 비우고 메인페이지를 보여주도록 구현했습니다.

<br/>

## 📌 구현 결과 (선택)

<br/>

## 📌 기타 사항

감사합니다.

<br/>

### CODE REVIEW

**Comment by JIMIN1020** [comment]
<h2>Coverage Report</h2>
<table> <thead> <tr> <th align="center">Status</th> <th align="left">Category</th> <th align="right">Percentage</th> <th align="right">Covered / Total</th> </tr> </thead> <tbody> <tr> <td align="center">🔵</td> <td align="left">Lines</td> <td align="right">0%</td> <td align="right">0 / 4405</td> </tr> <tr> <td align="center">🔵</td> <td align="left">Statements</td> <td align="right">0%</td> <td align="right">0 / 4405</td> </tr> <tr> <td align="center">🔵</td> <td align="left">Functions</td> <td align="right">3.65%</td> <td align="right">6 / 164</td> </tr> <tr> <td align="center">🔵</td> <td align="left">Branches</td> <td align="right">3.65%</td> <td align="right">6 / 164</td> </tr> </tbody> </table><details><summary>File Coverage</summary><table> <thead> <tr> <th align="left">File</th> <th align="right">Stmts</th> <th align="right">% Branch</th> <th align="right">% Funcs</th> <th align="right">% Lines</th> <th align="left">Uncovered Lines</th> </tr> </thead> <tbody> <tr> <td align="left" colspan="6"><b>Changed Files</b></td> </tr> <tr> <td align="left"><a href="https://github.com/Licruit/Licruit-frontend/blob/03310961bc54b95cfbc5fcbca73bb900cbb37bec/src/main.tsx">src/main.tsx</a></td> <td align="right">0%</td> <td align="right">0%</td> <td align="right">0%</td> <td align="right">0%</td> <td align="left"><a href="https://github.com/Licruit/Licruit-frontend/blob/03310961bc54b95cfbc5fcbca73bb900cbb37bec/src/main.tsx#L1-L3" class="text-red">1-3</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/03310961bc54b95cfbc5fcbca73bb900cbb37bec/src/main.tsx#L5-L20" class="text-red">5-20</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/03310961bc54b95cfbc5fcbca73bb900cbb37bec/src/main.tsx#L22" class="text-red">22</a></td> </tr> <tr> <td align="left"><a href="https://github.com/Licruit/Licruit-frontend/blob/03310961bc54b95cfbc5fcbca73bb900cbb37bec/src/features/Profile/api/signOut.ts">src/features/Profile/api/signOut.ts</a></td> <td align="right">0%</td> <td align="right">0%</td> <td align="right">0%</td> <td align="right">0%</td> <td align="left"><a href="https://github.com/Licruit/Licruit-frontend/blob/03310961bc54b95cfbc5fcbca73bb900cbb37bec/src/features/Profile/api/signOut.ts#L1" class="text-red">1</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/03310961bc54b95cfbc5fcbca73bb900cbb37bec/src/features/Profile/api/signOut.ts#L4-L7" class="text-red">4-7</a></td> </tr> <tr> <td align="left"><a href="https://github.com/Licruit/Licruit-frontend/blob/03310961bc54b95cfbc5fcbca73bb900cbb37bec/src/features/Profile/components/Mypage/MyPageSideMenu.tsx">src/features/Profile/components/Mypage/MyPageSideMenu.tsx</a></td> <td align="right">0%</td> <td align="right">0%</td> <td align="right">0%</td> <td align="right">0%</td> <td align="left"><a href="https://github.com/Licruit/Licruit-frontend/blob/03310961bc54b95cfbc5fcbca73bb900cbb37bec/src/features/Profile/components/Mypage/MyPageSideMenu.tsx#L1-L7" class="text-red">1-7</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/03310961bc54b95cfbc5fcbca73bb900cbb37bec/src/features/Profile/components/Mypage/MyPageSideMenu.tsx#L13-L14" class="text-red">13-14</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/03310961bc54b95cfbc5fcbca73bb900cbb37bec/src/features/Profile/components/Mypage/MyPageSideMenu.tsx#L16-L20" class="text-red">16-20</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/03310961bc54b95cfbc5fcbca73bb900cbb37bec/src/features/Profile/components/Mypage/MyPageSideMenu.tsx#L22-L33" class="text-red">22-33</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/03310961bc54b95cfbc5fcbca73bb900cbb37bec/src/features/Profile/components/Mypage/MyPageSideMenu.tsx#L35" class="text-red">35</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/03310961bc54b95cfbc5fcbca73bb900cbb37bec/src/features/Profile/components/Mypage/MyPageSideMenu.tsx#L47" class="text-red">47</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/03310961bc54b95cfbc5fcbca73bb900cbb37bec/src/features/Profile/components/Mypage/MyPageSideMenu.tsx#L63" class="text-red">63</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/03310961bc54b95cfbc5fcbca73bb900cbb37bec/src/features/Profile/components/Mypage/MyPageSideMenu.tsx#L66" class="text-red">66</a></td> </tr> <tr> <td align="left"><a href="https://github.com/Licruit/Licruit-frontend/blob/03310961bc54b95cfbc5fcbca73bb900cbb37bec/src/features/Profile/components/SignOut/CheckUser.tsx">src/features/Profile/components/SignOut/CheckUser.tsx</a></td> <td align="right">0%</td> <td align="right">0%</td> <td align="right">0%</td> <td align="right">0%</td> <td align="left"><a href="https://github.com/Licruit/Licruit-frontend/blob/03310961bc54b95cfbc5fcbca73bb900cbb37bec/src/features/Profile/components/SignOut/CheckUser.tsx#L1-L5" class="text-red">1-5</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/03310961bc54b95cfbc5fcbca73bb900cbb37bec/src/features/Profile/components/SignOut/CheckUser.tsx#L12-L13" class="text-red">12-13</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/03310961bc54b95cfbc5fcbca73bb900cbb37bec/src/features/Profile/components/SignOut/CheckUser.tsx#L15-L19" class="text-red">15-19</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/03310961bc54b95cfbc5fcbca73bb900cbb37bec/src/features/Profile/components/SignOut/CheckUser.tsx#L21-L23" class="text-red">21-23</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/03310961bc54b95cfbc5fcbca73bb900cbb37bec/src/features/Profile/components/SignOut/CheckUser.tsx#L25-L33" class="text-red">25-33</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/03310961bc54b95cfbc5fcbca73bb900cbb37bec/src/features/Profile/components/SignOut/CheckUser.tsx#L35-L56" class="text-red">35-56</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/03310961bc54b95cfbc5fcbca73bb900cbb37bec/src/features/Profile/components/SignOut/CheckUser.tsx#L58" class="text-red">58</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/03310961bc54b95cfbc5fcbca73bb900cbb37bec/src/features/Profile/components/SignOut/CheckUser.tsx#L60-L62" class="text-red">60-62</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/03310961bc54b95cfbc5fcbca73bb900cbb37bec/src/features/Profile/components/SignOut/CheckUser.tsx#L65" class="text-red">65</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/03310961bc54b95cfbc5fcbca73bb900cbb37bec/src/features/Profile/components/SignOut/CheckUser.tsx#L72" class="text-red">72</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/03310961bc54b95cfbc5fcbca73bb900cbb37bec/src/features/Profile/components/SignOut/CheckUser.tsx#L78" class="text-red">78</a></td> </tr> <tr> <td align="left"><a href="https://github.com/Licruit/Licruit-frontend/blob/03310961bc54b95cfbc5fcbca73bb900cbb37bec/src/features/Profile/components/SignOut/Complete.tsx">src/features/Profile/components/SignOut/Complete.tsx</a></td> <td align="right">0%</td> <td align="right">0%</td> <td align="right">0%</td> <td align="right">0%</td> <td align="left"><a href="https://github.com/Licruit/Licruit-frontend/blob/03310961bc54b95cfbc5fcbca73bb900cbb37bec/src/features/Profile/components/SignOut/Complete.tsx#L1-L3" class="text-red">1-3</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/03310961bc54b95cfbc5fcbca73bb900cbb37bec/src/features/Profile/components/SignOut/Complete.tsx#L9-L10" class="text-red">9-10</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/03310961bc54b95cfbc5fcbca73bb900cbb37bec/src/features/Profile/components/SignOut/Complete.tsx#L12-L15" class="text-red">12-15</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/03310961bc54b95cfbc5fcbca73bb900cbb37bec/src/features/Profile/components/SignOut/Complete.tsx#L17-L19" class="text-red">17-19</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/03310961bc54b95cfbc5fcbca73bb900cbb37bec/src/features/Profile/components/SignOut/Complete.tsx#L21" class="text-red">21</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/03310961bc54b95cfbc5fcbca73bb900cbb37bec/src/features/Profile/components/SignOut/Complete.tsx#L23-L30" class="text-red">23-30</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/03310961bc54b95cfbc5fcbca73bb900cbb37bec/src/features/Profile/components/SignOut/Complete.tsx#L32-L33" class="text-red">32-33</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/03310961bc54b95cfbc5fcbca73bb900cbb37bec/src/features/Profile/components/SignOut/Complete.tsx#L35" class="text-red">35</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/03310961bc54b95cfbc5fcbca73bb900cbb37bec/src/features/Profile/components/SignOut/Complete.tsx#L37" class="text-red">37</a></td> </tr> <tr> <td align="left"><a href="https://github.com/Licruit/Licruit-frontend/blob/03310961bc54b95cfbc5fcbca73bb900cbb37bec/src/features/Profile/components/SignOut/Confirm.tsx">src/features/Profile/components/SignOut/Confirm.tsx</a></td> <td align="right">0%</td> <td align="right">0%</td> <td align="right">0%</td> <td align="right">0%</td> <td align="left"><a href="https://github.com/Licruit/Licruit-frontend/blob/03310961bc54b95cfbc5fcbca73bb900cbb37bec/src/features/Profile/components/SignOut/Confirm.tsx#L1-L5" class="text-red">1-5</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/03310961bc54b95cfbc5fcbca73bb900cbb37bec/src/features/Profile/components/SignOut/Confirm.tsx#L11-L13" class="text-red">11-13</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/03310961bc54b95cfbc5fcbca73bb900cbb37bec/src/features/Profile/components/SignOut/Confirm.tsx#L15-L17" class="text-red">15-17</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/03310961bc54b95cfbc5fcbca73bb900cbb37bec/src/features/Profile/components/SignOut/Confirm.tsx#L19" class="text-red">19</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/03310961bc54b95cfbc5fcbca73bb900cbb37bec/src/features/Profile/components/SignOut/Confirm.tsx#L21-L24" class="text-red">21-24</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/03310961bc54b95cfbc5fcbca73bb900cbb37bec/src/features/Profile/components/SignOut/Confirm.tsx#L26-L30" class="text-red">26-30</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/03310961bc54b95cfbc5fcbca73bb900cbb37bec/src/features/Profile/components/SignOut/Confirm.tsx#L32-L34" class="text-red">32-34</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/03310961bc54b95cfbc5fcbca73bb900cbb37bec/src/features/Profile/components/SignOut/Confirm.tsx#L36" class="text-red">36</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/03310961bc54b95cfbc5fcbca73bb900cbb37bec/src/features/Profile/components/SignOut/Confirm.tsx#L38" class="text-red">38</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/03310961bc54b95cfbc5fcbca73bb900cbb37bec/src/features/Profile/components/SignOut/Confirm.tsx#L46-L48" class="text-red">46-48</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/03310961bc54b95cfbc5fcbca73bb900cbb37bec/src/features/Profile/components/SignOut/Confirm.tsx#L53" class="text-red">53</a></td> </tr> <tr> <td align="left"><a href="https://github.com/Licruit/Licruit-frontend/blob/03310961bc54b95cfbc5fcbca73bb900cbb37bec/src/features/Profile/components/SignOut/Reason.tsx">src/features/Profile/components/SignOut/Reason.tsx</a></td> <td align="right">0%</td> <td align="right">0%</td> <td align="right">0%</td> <td align="right">0%</td> <td align="left"><a href="https://github.com/Licruit/Licruit-frontend/blob/03310961bc54b95cfbc5fcbca73bb900cbb37bec/src/features/Profile/components/SignOut/Reason.tsx#L1-L5" class="text-red">1-5</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/03310961bc54b95cfbc5fcbca73bb900cbb37bec/src/features/Profile/components/SignOut/Reason.tsx#L11-L14" class="text-red">11-14</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/03310961bc54b95cfbc5fcbca73bb900cbb37bec/src/features/Profile/components/SignOut/Reason.tsx#L16-L20" class="text-red">16-20</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/03310961bc54b95cfbc5fcbca73bb900cbb37bec/src/features/Profile/components/SignOut/Reason.tsx#L22" class="text-red">22</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/03310961bc54b95cfbc5fcbca73bb900cbb37bec/src/features/Profile/components/SignOut/Reason.tsx#L24-L26" class="text-red">24-26</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/03310961bc54b95cfbc5fcbca73bb900cbb37bec/src/features/Profile/components/SignOut/Reason.tsx#L28" class="text-red">28</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/03310961bc54b95cfbc5fcbca73bb900cbb37bec/src/features/Profile/components/SignOut/Reason.tsx#L31-L48" class="text-red">31-48</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/03310961bc54b95cfbc5fcbca73bb900cbb37bec/src/features/Profile/components/SignOut/Reason.tsx#L50-L51" class="text-red">50-51</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/03310961bc54b95cfbc5fcbca73bb900cbb37bec/src/features/Profile/components/SignOut/Reason.tsx#L53" class="text-red">53</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/03310961bc54b95cfbc5fcbca73bb900cbb37bec/src/features/Profile/components/SignOut/Reason.tsx#L55" class="text-red">55</a></td> </tr> <tr> <td align="left"><a href="https://github.com/Licruit/Licruit-frontend/blob/03310961bc54b95cfbc5fcbca73bb900cbb37bec/src/features/Profile/components/SignOut/SignOut.tsx">src/features/Profile/components/SignOut/SignOut.tsx</a></td> <td align="right">0%</td> <td align="right">0%</td> <td align="right">0%</td> <td align="right">0%</td> <td align="left"><a href="https://github.com/Licruit/Licruit-frontend/blob/03310961bc54b95cfbc5fcbca73bb900cbb37bec/src/features/Profile/components/SignOut/SignOut.tsx#L1-L11" class="text-red">1-11</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/03310961bc54b95cfbc5fcbca73bb900cbb37bec/src/features/Profile/components/SignOut/SignOut.tsx#L17-L23" class="text-red">17-23</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/03310961bc54b95cfbc5fcbca73bb900cbb37bec/src/features/Profile/components/SignOut/SignOut.tsx#L25-L36" class="text-red">25-36</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/03310961bc54b95cfbc5fcbca73bb900cbb37bec/src/features/Profile/components/SignOut/SignOut.tsx#L38" class="text-red">38</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/03310961bc54b95cfbc5fcbca73bb900cbb37bec/src/features/Profile/components/SignOut/SignOut.tsx#L40-L71" class="text-red">40-71</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/03310961bc54b95cfbc5fcbca73bb900cbb37bec/src/features/Profile/components/SignOut/SignOut.tsx#L73" class="text-red">73</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/03310961bc54b95cfbc5fcbca73bb900cbb37bec/src/features/Profile/components/SignOut/SignOut.tsx#L75" class="text-red">75</a></td> </tr> <tr> <td align="left"><a href="https://github.com/Licruit/Licruit-frontend/blob/03310961bc54b95cfbc5fcbca73bb900cbb37bec/src/features/Profile/components/common/CancelAndNext.tsx">src/features/Profile/components/common/CancelAndNext.tsx</a></td> <td align="right">0%</td> <td align="right">0%</td> <td align="right">0%</td> <td align="right">0%</td> <td align="left"><a href="https://github.com/Licruit/Licruit-frontend/blob/03310961bc54b95cfbc5fcbca73bb900cbb37bec/src/features/Profile/components/common/CancelAndNext.tsx#L1-L3" class="text-red">1-3</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/03310961bc54b95cfbc5fcbca73bb900cbb37bec/src/features/Profile/components/common/CancelAndNext.tsx#L11-L12" class="text-red">11-12</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/03310961bc54b95cfbc5fcbca73bb900cbb37bec/src/features/Profile/components/common/CancelAndNext.tsx#L14-L22" class="text-red">14-22</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/03310961bc54b95cfbc5fcbca73bb900cbb37bec/src/features/Profile/components/common/CancelAndNext.tsx#L24-L33" class="text-red">24-33</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/03310961bc54b95cfbc5fcbca73bb900cbb37bec/src/features/Profile/components/common/CancelAndNext.tsx#L35-L36" class="text-red">35-36</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/03310961bc54b95cfbc5fcbca73bb900cbb37bec/src/features/Profile/components/common/CancelAndNext.tsx#L38" class="text-red">38</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/03310961bc54b95cfbc5fcbca73bb900cbb37bec/src/features/Profile/components/common/CancelAndNext.tsx#L40" class="text-red">40</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/03310961bc54b95cfbc5fcbca73bb900cbb37bec/src/features/Profile/components/common/CancelAndNext.tsx#L46" class="text-red">46</a></td> </tr> <tr> <td align="left"><a href="https://github.com/Licruit/Licruit-frontend/blob/03310961bc54b95cfbc5fcbca73bb900cbb37bec/src/features/Profile/constants/options.ts">src/features/Profile/constants/options.ts</a></td> <td align="right">0%</td> <td align="right">0%</td> <td align="right">0%</td> <td align="right">0%</td> <td align="left"><a href="https://github.com/Licruit/Licruit-frontend/blob/03310961bc54b95cfbc5fcbca73bb900cbb37bec/src/features/Profile/constants/options.ts#L1-L9" class="text-red">1-9</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/03310961bc54b95cfbc5fcbca73bb900cbb37bec/src/features/Profile/constants/options.ts#L11-L17" class="text-red">11-17</a></td> </tr> <tr> <td align="left"><a href="https://github.com/Licruit/Licruit-frontend/blob/03310961bc54b95cfbc5fcbca73bb900cbb37bec/src/features/Profile/hooks/useProfileMutation.ts">src/features/Profile/hooks/useProfileMutation.ts</a></td> <td align="right">0%</td> <td align="right">0%</td> <td align="right">0%</td> <td align="right">0%</td> <td align="left"><a href="https://github.com/Licruit/Licruit-frontend/blob/03310961bc54b95cfbc5fcbca73bb900cbb37bec/src/features/Profile/hooks/useProfileMutation.ts#L1-L2" class="text-red">1-2</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/03310961bc54b95cfbc5fcbca73bb900cbb37bec/src/features/Profile/hooks/useProfileMutation.ts#L5-L12" class="text-red">5-12</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/03310961bc54b95cfbc5fcbca73bb900cbb37bec/src/features/Profile/hooks/useProfileMutation.ts#L14" class="text-red">14</a></td> </tr> <tr> <td align="left"><a href="https://github.com/Licruit/Licruit-frontend/blob/03310961bc54b95cfbc5fcbca73bb900cbb37bec/src/features/Profile/hooks/useSignOutMutation.ts">src/features/Profile/hooks/useSignOutMutation.ts</a></td> <td align="right">0%</td> <td align="right">0%</td> <td align="right">0%</td> <td align="right">0%</td> <td align="left"><a href="https://github.com/Licruit/Licruit-frontend/blob/03310961bc54b95cfbc5fcbca73bb900cbb37bec/src/features/Profile/hooks/useSignOutMutation.ts#L1-L2" class="text-red">1-2</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/03310961bc54b95cfbc5fcbca73bb900cbb37bec/src/features/Profile/hooks/useSignOutMutation.ts#L5-L9" class="text-red">5-9</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/03310961bc54b95cfbc5fcbca73bb900cbb37bec/src/features/Profile/hooks/useSignOutMutation.ts#L11" class="text-red">11</a></td> </tr> <tr> <td align="left"><a href="https://github.com/Licruit/Licruit-frontend/blob/03310961bc54b95cfbc5fcbca73bb900cbb37bec/src/features/Profile/model/signout.model.ts">src/features/Profile/model/signout.model.ts</a></td> <td align="right">100%</td> <td align="right">100%</td> <td align="right">100%</td> <td align="right">100%</td> <td align="left"></td> </tr> <tr> <td align="left"><a href="https://github.com/Licruit/Licruit-frontend/blob/03310961bc54b95cfbc5fcbca73bb900cbb37bec/src/hooks/logout/useLogOut.ts">src/hooks/logout/useLogOut.ts</a></td> <td align="right">0%</td> <td align="right">0%</td> <td align="right">0%</td> <td align="right">0%</td> <td align="left"><a href="https://github.com/Licruit/Licruit-frontend/blob/03310961bc54b95cfbc5fcbca73bb900cbb37bec/src/hooks/logout/useLogOut.ts#L1-L3" class="text-red">1-3</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/03310961bc54b95cfbc5fcbca73bb900cbb37bec/src/hooks/logout/useLogOut.ts#L5-L7" class="text-red">5-7</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/03310961bc54b95cfbc5fcbca73bb900cbb37bec/src/hooks/logout/useLogOut.ts#L10-L15" class="text-red">10-15</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/03310961bc54b95cfbc5fcbca73bb900cbb37bec/src/hooks/logout/useLogOut.ts#L17-L18" class="text-red">17-18</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/03310961bc54b95cfbc5fcbca73bb900cbb37bec/src/hooks/logout/useLogOut.ts#L20" class="text-red">20</a></td> </tr> </tbody> </table></details>
<em>Generated in workflow <a href=https://github.com/Licruit/Licruit-frontend/actions/runs/10316072295>#83</a></em>

File: src/features/Profile/components/SignOut/SignOut.tsx [hunk 1]
```diff
@@ -2,14 +2,23 @@ import { useFunnel } from '@/hooks/form/useFunnel';
...
 import Complete from './Complete';
+import useSignOutMutaion from '../../hooks/useSignOutMutation';
 
-function SignOut() {
+interface Props {
+  onClose: () => void;
+}
+
+function SignOut({ onClose }: Props) {
+  const [signOutReason, setSignOutReason] = useState('');
```
Comment (JIMIN1020) [inline src/features/Profile/components/SignOut/SignOut.tsx:18]: 얘만 useState로 관리할 필요가 있을까요? `Reason` 컴포넌트 내부에서는 react hook form으로 관리하고 그걸 여기로 올려주는 것 같은데, 그냥 이 Funnel 전체를 react-hook-form으로 감싸서 데이터를 한 곳에서 관리할 수 있지 않을까요?? onNext에 저렇게 각각 함수를 넣어줄 필요 없이 handleSubmit으로 한번에 처리할 수 있을 것 같아요

**Review by JIMIN1020 — APPROVED** [review]
수고하셨습니다!!

File: src/features/Profile/components/SignOut/SignOut.tsx (hunk 1 above)
Comment (SebellKo) [reply src/features/Profile/components/SignOut/SignOut.tsx:18]: 그 방법이 덜 복잡하긴 할것 같네요.. 확인해보고 수정해 보겠습니다 !

File: src/features/Profile/components/SignOut/CheckUser.tsx [hunk 2]
```diff
@@ -1,49 +1,78 @@
...
+              {...register('companyNumber', { required: true })}
+            />
+            <FormInput
+              type='password'
+              placeholder='비밀번호를 입력해주세요'
+              hasVisibility
+              {...register('password', { required: true })}
+            />
+          </InputWrapper>
+          <Error className='error'>
```
Comment (juhee067) [inline src/features/Profile/components/SignOut/CheckUser.tsx:48]: 궁금한게 hook-form에 있는 에러 기능말고 prop로 사용하는 게 더 좋나요?!

File: src/features/Profile/components/SignOut/Confirm.tsx [hunk 3]
```diff
@@ -24,6 +24,8 @@ function Confirm({ onNext }: Props) {
         onClick={() => setIsChecked((prev) => !prev)}
       >
```
Comment (juhee067) [inline src/features/Profile/components/SignOut/Confirm.tsx:25]: 이 내용과는 별개로 이 파일 안에 
```
    탈퇴 후에는 사업자 등록번호 0000-00-0000으로 재가입 불가능하며,
        <br />
        해당 사업자 등록번호와 데이터는 복구되지 않습니다.
```
 이렇게 된 부분이 있떠라구요  이거 br 대신 \n 사용해서 스타일 속성에 whiteSpace인가? 설정하면 줄바꿈 되더라구요! 
접근성 때문인가? br을 사용하지 말라는 걸 본 적이 있는 거 같아요!

File: src/features/Profile/constants/options.ts [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+  '기타',
+];
+
+export const COMPANY_OPTIONS = [
+  '수익성이 좋은편이 아닌거같아요.',
+  '사용하기 불편해요.',
+  '결제 시스템이 없어서 불편해요.',
+  '거래조건이 안좋아요.',
+  '기타',
+];
```
Comment (juhee067) [inline src/features/Profile/constants/options.ts:17]: 궁금한 게 저희 기타 선택했을 때 입력하는 창을 또 만들어줘야하나요?

**Review by juhee067 — APPROVED** [review]
붸리붸리 굿!

File: src/features/Profile/components/SignOut/SignOut.tsx (hunk 1 above)
Comment (SebellKo) [reply src/features/Profile/components/SignOut/SignOut.tsx]: 수정완료 !

File: src/features/Profile/components/SignOut/Confirm.tsx (hunk 3 above)
Comment (SebellKo) [reply src/features/Profile/components/SignOut/Confirm.tsx:25]: 완전 꿀팁 !!

차마.. 전혀 몰랐던 사실입니다 !
감사합니다 ~

File: src/features/Profile/constants/options.ts (hunk 4 above)
Comment (SebellKo) [reply src/features/Profile/constants/options.ts:17]: 기타 선택시 추가 인풋을 추가하는게 좋을것 같습니다 !!

멘토님도 해당내용에 대해서 인풋을 추가하는게 좋을것 같다고 조언해 주셨어요 !

Comment (juhee067) [reply src/features/Profile/constants/options.ts:17]: 세종님 할 일도 추가!

File: src/features/Profile/components/SignOut/Confirm.tsx (hunk 3 above)
Comment (ksy90101) [reply src/features/Profile/components/SignOut/Confirm.tsx:25]: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/br#accessibility

정확히는 p 태그를 쓰는게 더 좋다고 하죠.

```
<br> 태그를 사용하여 별도의 단락을 만드는 것은 좋지 않은 관행일 뿐만 아니라, 스크린 리더 기술을 사용하는 사람들에게 문제가 될 수 있습니다. 스크린 리더는 <br> 요소의 존재를 알릴 수는 있지만, 그 안에 포함된 내용을 제대로 읽지 못할 수 있습니다. 이는 스크린 리더를 사용하는 사람들에게 혼란스럽고 좌절감을 줄 수 있습니다.

따라서 <p> 요소를 사용하고, CSS 속성인 margin을 사용하여 단락 간의 간격을 조절하는 것이 좋습니다.
```

Comment (SebellKo) [reply src/features/Profile/components/SignOut/Confirm.tsx:25]: p 태그 확인했습니다.. !!
div 태그를 사용하고 있었네요..

**Pull request merged**